### PR TITLE
Adding AWS Profile block to display the current `AWS_PROFILE`

### PR DESCRIPTION
**Pull request by dsiguero**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,14 @@ The following sections describe the built-in blocks and their customizing option
 
 ### Built-in blocks
 
+#### `awsprofile`
+
+Displays the current AWS profile (from the `$AWS_PROFILE` env variable).
+
+| Variable | Default | Meaning |
+| :------- | :------ | :------ |
+| `BLOX_BLOCK__AWSPROFILE_COLOR` | `'orange'` | This block's color |
```
Comment (yardnsm) [inline README.md]:
```suggestion
| `BLOX_BLOCK__AWSPROFILE_COLOR` | `'172' (orange)` | This block's color |
```

Reflect the actual given value

File: blox.zsh [hunk 2]
```diff
@@ -17,6 +17,7 @@ autoload -Uz promptinit && promptinit
 
 source "$__BLOX_ROOT/src/utils.zsh"
 
+source "$__BLOX_ROOT/src/blocks/awsprofile.zsh"
```
Comment (yardnsm) [inline blox.zsh]: Include this block after the inclusion of `virtualenv` block

File: README.md [hunk 3]
```diff
@@ -79,6 +79,14 @@ The following sections describe the built-in blocks and their customizing option
 
 ### Built-in blocks
 
+#### `awsprofile`
```
Comment (yardnsm) [inline README.md]: As this is a new block, it should be listed last.

File: src/segments.zsh [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
 # Defualts
 __BLOX_SEG_DEFAULT__UPPER_LEFT=( host cwd git exec_time )
-__BLOX_SEG_DEFAULT__UPPER_RIGHT=( bgjobs nodejs pyenv virtualenv time )
+__BLOX_SEG_DEFAULT__UPPER_RIGHT=( bgjobs nodejs pyenv virtualenv awsprofile )
```
Comment (yardnsm) [inline src/segments.zsh]:
```suggestion
__BLOX_SEG_DEFAULT__UPPER_RIGHT=( bgjobs nodejs pyenv virtualenv awsprofile time )
```

Why override the `time` block? I put it between the `virtualenv` and the `time` blocks.

**Review by yardnsm — CHANGES_REQUESTED** [review]
Sorry for the long delay, I got a lot of work lately. Anyway, looking good! Just some minor requests.

**Comment by dsiguero** [comment]
No worries, I can definitely relate :) please let me know if there are any other required changes.

**Comment by yardnsm** [comment]
Looks great, thanks!

**Pull request merged**